### PR TITLE
use assertElementNotNull method to check if element is null

### PR DESCRIPTION
**Pull request by individual-it**
## Description
use a new method to check if a `find()` function returned `null` or an actual element

## Motivation and Context
- DRY
- delete code

## How Has This Been Tested?

- run some UI tests locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/lib/OwncloudPage.php [hunk 1]
```diff
@@ -768,4 +761,18 @@ public function quotedText($text) {
...
+
+	/**
+	 *
+	 * @param NodeElement $element
+	 * @param string $message
+	 *
+	 * @throws ElementNotFoundException
+	 * @return void
+	 */
+	public function assertElementNotNull($element, $message) {
```
Comment (phil-davis) [inline tests/acceptance/features/lib/OwncloudPage.php]: Suggest changing all calls to this method to instead call:
```
PHPUnit_Framework_Assert::assertNotNull($var, "message");
```

**Review by phil-davis — COMMENTED** [review]
Just 1 search/replace and delete of method.

File: tests/acceptance/features/lib/OwncloudPage.php (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/lib/OwncloudPage.php]: Actually there are core and app tests that already want to sometimes catch these exceptions. So it is not so great to use ``PHPUnit_Framework_Assert::assertNotNull()``

**Review by phil-davis — APPROVED** [review]

**Comment by individual-it** [comment]
backport in #33272

**Pull request merged**